ci/scripts/build.sh

```diff
@@ -2,9 +2,9 @@
 
 cwd=$(pwd)
 
-export GOPATH=$cwd/go
+export GOPATH=$cwd
```

Comment:
it's probably not very important, but maybe we should use $cwd directly in pushd, because we are no longer using GOPATH and defining this var might be confusing.